Sources/Overload/OvTools/include/OvTools/Utils/String.h

```diff
@@ -18,6 +18,13 @@ namespace OvTools::Utils
 	class String
 	{
 	public:
+		enum ETrimType
+		{
+			Leading = 1,
+			Trailing = 2,
+			TwoWays = 4 // Leading and Trailing
```

Comment:
Could be set to 3, so that `ETrimType::TwoWays == ETrimType::Leading & ETrimType::Trailing`